### ISSUE TITLE
Documentation includes legacy Hydrogen script h2-compile

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#3840]
## Description
The repository includes references to the Hydrogen script `h2-compile` that is [no longer part of version 2.0.0](https://github.com/hydrogen-css/hydrogen/blob/release/2.0.0/package.json#L13-L22).

## Acceptance Criteria
- [ ] remove any references to `h2-compile` (excluding _tc-report_ that still uses hydrogen version 1.1.12)

**Comment by mnigh** [comment]
@substrae is this issue still accurate?

**Comment by substrae** [comment]
Yep - there's `h2-build` and `h2-watch`, so compile won't be helpful.

**Comment by mnigh** [comment]
`h2-compile` is no longer referenced in the codebase aside from references in **tc-report** that still uses hydrogen version 1.1.12.

## Screenshot
![Screen Shot 2023-03-02 at 14 11 31](https://user-images.githubusercontent.com/3046459/222528234-fea5bad3-db57-4690-9dc9-b40269ebf0ce.png)
